master/internal/job/job.go

```diff
@@ -135,6 +147,11 @@ func (j *Manager) GetJobs(
 		return jobsInRM[i].JobId < jobsInRM[j].JobId
 	})
 
+	// Append any Non-DAI jobs at the bottom of the list.
+	if len(nonDaiJobs) > 0 {
+		jobsInRM = append(jobsInRM, nonDaiJobs...)
+	}
```

Comment:
comment doesn't really tell us anything new. also, you can just do
```suggestion
	jobsInRM = append(jobsInRM, nonDaiJobs...)
```
because appending a nil slice is valid and a noop